docs/source/docs/visualisation_tool.md

```diff
@@ -0,0 +1,158 @@
+# Visualisation tool
+
+DeepReg provides a visuaisation tool which allows the user to generate various
+visualisations from nifti images. The tool is compatible with outputs from
+deepreg_predict as well as with other nifti images.
+
+The visualisation tool currently offers four functionalities using the command
+`deepreg_vis`, with their Python functions explained in the final section this document.
+Runn the below examples after changing directory to the root folder `DeepReg`.
+
```

Comment:
Typos here : 
"in the final section **of** this document"
"**Run**"
"examples below"

Reply:
fixed